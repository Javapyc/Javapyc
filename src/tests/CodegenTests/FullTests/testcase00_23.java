// Not an official testcase; causes integer overflow

class Main {
    public static void main(String[] args){
        int a = 2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2;
	int b = 9*9*9*9*9*9*9*0*9*9*9*9*9*9;
	System.out.println(a);
	System.out.println(b);
	System.out.println(a / 2);
    }
}