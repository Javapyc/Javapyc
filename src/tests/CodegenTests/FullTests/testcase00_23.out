-2147483648
0
-1073741824
